scripts/gen-i18n.js

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/env node
```

Comment:
might be beneficial to extract this to a shared place somewhere given that element-web & react-sdk already share react-sdk's feels a bit weird to have a 2nd copy of it and it might miss updates we end up doing to the other one.